tools/cldr/cldr-to-icu/README.txt

```diff
@@ -0,0 +1,71 @@
+Basic instructions for running the LdmlConverter via Maven
+----------------------------------------------------------
+
+Note that these instructions do not currently support configuration of the converter for things
+such as limiting the set of files produced. That is supported in code and could be easily added
+to the binary, or encapsulated via an Ant task, but currently it is not directly supported.
+See the IcuConverterConfig class for the API by which this can be supported.
+
+
+Notes on JDK versions:
+----------------------
+
+It is possible that some JDK versions will not, by default, support the set of features necessary
```

Comment:
From maven we should be able to specify the exact JDK version used, so is this statement still needed?

Reply:
This is a "coded" bit of advice that's really for Googlers. Any Google setup uses an internal JDK by default which has had the SAX parser's behaviour limited for security reasons. The errors are weird and non-obvious and have taken me, and others, days to track down. The fix is just to use the non-production JDK that's also available (but not the default).

I guess the error message is pretty reasonable now on its own, so maybe this isn't required in the actual readme any more...